gulpfile.js

```diff
@@ -107,9 +108,9 @@ async function processTypesFile() {
 }
 
 async function concatLfCdnToScript() {
-  src([LF_CDN_MAINJS_FILEPATH, LF_CDN_RUNTIMEJS_FILEPATH, LF_CDN_POLYFILLSJS_FILEPATH])
+  src([LF_CDN_MAINJS_FILEPATH, LF_CDN_RUNTIMEJS_FILEPATH])
   .pipe(concat(SCRIPT_FILE))
```

Comment:
why is this no longer needed?

Reply:
because we import zone.js in polyfills, so we can exclude zone.js by not concating polyfills to the final cdn script

Reply:
It was because ng test failed with zone removed from polyfills, the error message says that "zone is not defined in vendor.js". So I was assuming we needed zone to be present in polyfills.js but I can look more into it